sdks/python/apache_beam/runners/dataflow/internal/names.py

```diff
@@ -36,10 +36,10 @@
 
 # Update this version to the next version whenever there is a change that will
 # require changes to legacy Dataflow worker execution environment.
-BEAM_CONTAINER_VERSION = 'beam-master-20211202'
+BEAM_CONTAINER_VERSION = 'beam-master-20211213'
 # Update this version to the next version whenever there is a change that
 # requires changes to SDK harness container or SDK harness launcher.
-BEAM_FNAPI_CONTAINER_VERSION = 'beam-master-20211202'
+BEAM_FNAPI_CONTAINER_VERSION = 'beam-master-20211213'
```

Comment:
given that python changes are likely to not happen on 20211213, I think I would prefer not changing this since the path for discovering what a beam-master bump was for can be confusing and it would be better not to suggest this PR changes the OS used

Reply:
The CL with python changes is cherry-picked on 20211213. We can use this tag once the process completes